From 1f339f811c246542fa163daa116f27e8ec48683d Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Mon, 15 May 2017 16:09:50 +0300
Subject: [PATCH 1019/1345] fix: crypto: inside-secure: remove useless address
 from invalidation

commit  c9f504a4da970195ce390715201558bc37a8176a from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

invalidation request does not need to set the result descriptor
address, only the command descriptor uses this address for the
invalidation.

this change does not have any functional impact, only alignes the code
with eip197 invalidation command requirements.

Change-Id: I4e3a329274542700d4a4a13f19af13623f381c0b
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39483
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39533
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/inside-secure/safexcel.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index 5f0a7db..d111384 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -863,7 +863,7 @@ int safexcel_invalidate_cache(struct crypto_async_request *async,
 				       CONTEXT_CONTROL_HW_SERVICES_OFFSET;
 
 	/* prepare result descriptor */
-	rdesc = safexcel_add_rdesc(priv, ring, true, true, ctxr_dma, 0);
+	rdesc = safexcel_add_rdesc(priv, ring, true, true, 0, 0);
 
 	if (IS_ERR(rdesc)) {
 		ret = PTR_ERR(rdesc);
-- 
1.7.9.5

